ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/SqlLedgerReaderWriter.scala

```diff
@@ -55,7 +56,6 @@ final class SqlLedgerReaderWriter(
     latestSequenceNo => dispatcher.signalNewHead(latestSequenceNo + 1),
   )
 
-  // TODO: implement
   override def currentHealth(): HealthStatus = Healthy
```

Comment:
Such TODOs should be tracked in an issue tracker. I.e., we need to introduce health checking for SQL-based ledgers which obviously depends on the backing server implementation.

Reply:
Still, I don't think this TODO removal belongs here; it would be better to have a separate one after creating the tracking issue that removes all health-related TODOs

Reply:
@gerolf-da Do you want to create an issue to track this, please?

Reply:
Already exists: #4224

Reply:
I think it's fine to remove this; didn't have to be in this PR but there's no harm.

Reply:
Thanks for the confirmation, @SamirTalwar-DA